Examples/Run/Reconstruction/CMakeLists.txt

```diff
@@ -18,3 +18,24 @@ target_link_libraries(
 install(
   TARGETS ActsRecTruthTracks
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+add_executable(
+    ActsRecCKFTracks
+    RecCKFTracks.cpp)
+target_link_libraries(
+    ActsRecCKFTracks
+    PRIVATE
+    ActsExamplesFramework
+    ActsExamplesCommon
+    ActsExamplesDigitization
+    ActsExamplesTrackFinding
+    ActsExamplesDetectorGeneric
+    ActsExamplesMagneticField
+    ActsExamplesIoCsv
+    ActsExamplesIoPerformance)
+
+install(
+  TARGETS
+   ActsRecTruthTracks
```

Comment:
@XiaocongAi Shouldn't this have been ActsRecCKFTracks?

Reply:
You're absolutely right. It has been already fixed in another PR (which I cannot remember...).

Reply:
I see, thanks. I was reviewing the 0.25 build system changes while adding that version to spack, and this line surprised me.